tests/components/test_nuheat.py

```diff
@@ -0,0 +1,44 @@
+"""NuHeat component tests."""
+import unittest
+
+from unittest.mock import patch
+from tests.common import get_test_home_assistant, MockDependency
+
+from homeassistant.components import nuheat
+
+VALID_CONFIG = {
+    "nuheat": {
+        "username": "warm",
+        "password": "feet",
+        "devices": "thermostat123"
+    }
+}
+
+
+class TestNuHeat(unittest.TestCase):
+    """Test the NuHeat component."""
+
+    def setUp(self):
+        """Initialize the values for this test class."""
+        self.hass = get_test_home_assistant()
+        self.config = VALID_CONFIG
+
+    def tearDown(self):
+        """Teardown this test class. Stop hass."""
+        self.hass.stop()
+
+    @MockDependency("nuheat")
+    @patch("homeassistant.helpers.discovery.load_platform")
+    def test_setup(self, mocked_nuheat, mocked_load):
```

Comment:
I think the order of arguments is inverted.

Reply:
I thought so too at first, but apparently the `MockDependency` helper does things a bit differently.